content-elements/form/audiorecorder/prototype/index.js

```diff
@@ -0,0 +1,19 @@
+require('./styles.scss');
+const {cx, Icon} = require('@bsi-cx/design-build');
+
+module.exports = (
+  template = require('../template.twig'),
+  elementId = 'audiorecorder-e89a2a',
+  elementLabel = 'Audio Recorder',
+  formTextPartId = 'id-file-input-c1db98',
+  formTextPartLabel = 'Field ID',
+) => cx
+  .contentElement
+  .withFile(template)
+  .withElementId(elementId)
+  .withLabel(elementLabel)
```

Comment:
You also need to choose an icon for the element:
<img width="239" height="129" alt="image" src="https://github.com/user-attachments/assets/1d4d7929-3e83-4ec4-8621-587f7d701aaf" />

```suggestion
  .withLabel(elementLabel)
  .withIcon(Icon.MEGAPHONE)
```